vault/dendron.tutorial.taking-notes.md

```diff
@@ -1,54 +1,97 @@
 
-> 🌱 To create a hierarchy, bring up lookup again `(Ctrl+L / Cmd+L)`. Type `recipes.vegetarian` and then press enter.
+You've created your first hierarchy! **[[Hierarchies|dendron://dendron.dendron-site/dendron.topic.hierarchies]]** in Dendron are created by having a `.` (a "period" or "dot") delimiter in the file name. For more information on how this works: [[What's with the .'s?|dendron://dendron.dendron-site/dendron.topic.hierarchies#whats-with-the-s]]
 
-You have now created your first hierarchy. Hierarchies in Dendron are created by having a `.` delimiter in the file name. Take a look at the tree view in the side panel to see that the `vegetarian` note exists under the `recipes` hierarchy. You can also see in the Workspace panel that the note file is stored as `recipes.vegetarian.md`.
+Take a look at the Tree View to seethe `vegetarian` note under the `recipes` hierarchy. You can also see the the note stored as `recipes.vegetarian.md` in the File Explorer Pane.
 
-You can create a hierarchy at any level:
+You can create a hierarchy at any level.
 
-> 🌱 Type the following into lookup and hit enter: `recipes.italian.desserts.tiramisu`
+> 🌱 **ACTION:** Type the following into lookup and hit enter: `recipes.italian.desserts.tiramisu`
 
-You'll notice in the tree view that there is now a `+` sign next to `italian` and `desserts`. The plus sign indicates that this note is a stub. A stub is a placeholder for a note that hasn't actually been created. Dendron uses stubs to avoid cluttering your file system with empty notes when creating hierarchies.
+You'll there is a `+` sign next to `italian` and `desserts` when viewing in the lookup or Tree View. The plus sign indicates that this note is a stub. A **[[stub|dendron://dendron.dendron-site/dendron.concepts#stubs]]** is a placeholder for a note that hasn't been created. Dendron uses stubs to avoid cluttering your file system with empty notes when creating hierarchies.
 
 ![Tree View](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-tree-view.png)
 
-Following this recipes example, you can try creating other notes that might be relevant. Some examples: `recipes.italian.appetizers`, `recipes.chinese`, `recipes.chinese.appetizers`, `recipes.ingredients.favorites`, `recipes.ingredients.shopping-list`. Hierarchies are flexible, and you can create them in whichever way works best for you. Dendron allows you to easily change the hierarchies later if you change your mind on how to organize your notes.
+Following this recipes example, you can create other related. Some examples:
+
+```
+recipes.italian.appetizers
+recipes.chinese
+recipes.chinese.appetizers
+recipes.ingredients.favorites
+recipes.ingredients.shopping-list
+```
+
+Hierarchies are flexible, and you can create them in whichever way works best for you. Dendron allows you to [[easily change the hierarchies later|dendron://dendron.dendron-site/dendron.topic.refactoring]] if you change your mind on how to organize your notes.
```

Comment:
now that we talked about hiearchies, add note reference to `why hiearchy` with an intro. (eg. you might wonder why Dendron uses hiearchies...)